test/contlinear.jl

```diff
@@ -292,6 +292,67 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
         end
     end
 
+    @testset "addvariable! (one by one) interface" begin
+        # Min -x
+        # s.t. x + y <= 1
+        # x, y >= 0
+
+        @test MOI.supportsproblem(solver, MOI.ScalarAffineFunction, [(MOI.ScalarAffineFunction{Float64},MOI.LessThan),(MOI.ScalarVariablewiseFunction,MOI.GreaterThan)])
```

Comment:
Wait, you mean we should not add `{Float64}` in `MOI.LessThan` in `MOI.supportsproblem`?